Makefile

```diff
@@ -609,6 +609,10 @@ e2e-parallel: e2e-set-image prep-e2e ## Run non-destructive end-to-end tests con
 e2e-serial: e2e-set-image prep-e2e ## Run destructive end-to-end tests serially.
 	@CONTENT_IMAGE=$(E2E_CONTENT_IMAGE_PATH) BROKEN_CONTENT_IMAGE=$(E2E_BROKEN_CONTENT_IMAGE_PATH) $(GO) test ./tests/e2e/serial $(E2E_GO_TEST_FLAGS) -args $(E2E_ARGS) | tee tests/e2e-test.log
 
+.PHONY: e2e-rosa
+e2e-rosa: e2e-set-image prep-e2e ## Run non-destructive end-to-end tests concurrently.
+	@CONTENT_IMAGE=$(E2E_CONTENT_IMAGE_PATH) BROKEN_CONTENT_IMAGE=$(E2E_BROKEN_CONTENT_IMAGE_PATH) $(GO) test ./tests/e2e/rosa $(E2E_GO_TEST_FLAGS) -args $(E2E_ARGS) --platform rosa | tee tests/e2e-test.log
```

Comment:
we could also reuse PLATFORM environment, https://github.com/ComplianceAsCode/compliance-operator/pull/515/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R7

Reply:
Aha - I think I see what you mean. Let me try that.

Reply:
I think the only difference here would be how you invoke the tests, either `make e2e-rosa` or `PLATFORM=rosa make e2e-rosa`.

If we set `--platform $(PLATFORM)` we need to make sure we specify the platform prior to calling the target, otherwise the runner will assume it's openshift, and fail on MachineConfigPool configuration.

I think I lean towards `make e2e-rosa` since it's less for the caller to specify.

What do you think?

Reply:
I think that makes senses, we can set the environment later as well